scripts/copy-css.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+mkdir -p dist/assets dist/css/v2 dist/scss/v2
+
+cp -r node_modules/@stream-io/stream-chat-css/dist/assets/* dist/assets
+cp -r node_modules/@stream-io/stream-chat-css/dist/css/*.css dist/css
```

Comment:
The `master` branch does not count with legacy styles anymore. Only `dist/v2/*` will be available.